licenses.d2iq.yaml

```diff
@@ -5,6 +5,8 @@ ignore:
   - docker.io/bitnami/external-dns:0.13.6-debian-11-r11
   - docker.io/bitnami/postgresql:11.16.0-debian-11-r9
   - docker.io/bitnami/postgresql:15.2.0-debian-11-r21
+  - docker.io/bitnami/postgresql:15.4.0-debian-11-r44
```

Comment:
is it possible to use the same postgresql image as gitea here? this will keep the bundle size smaller and keeps cve's minimal in future.

Reply:
cc @bcm820 who worked on removing duplicate images recently may have more inputs here on which image to use here.

Reply:
I think we can. I'll change it.

Reply:
Done.